README.md

```diff
@@ -42,16 +45,6 @@ make up
 As part of `make up`, `./fixtures.sh` will run and create the initial user and
 organization you can log in with.
```

Comment:
I am still missing two pieces of info:

1. The docker-compose stuff happens in the `testing` sub folder of giantswarm/api
2. You have to run `make aws|azure|kvm` to create a docke-compose.yaml file.

Reply:
Ok! adding that.

We do have a `cd` command as part of the example. but i'll add the `make aws` part there.